src/Behavior/DataProxyMutableBehavior.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Focus\Data\Behavior;
+
+use Focus\Data\Data;
+use Focus\Data\KeyedData\KeyedDataObject;
+
+/**
+ * Provides simple source() method for a DataProxyMutable extension
+ *
+ * @see \Focus\Data\DataProxyMutable
+ */
+trait DataProxyMutableBehavior
+{
+    public function __construct(
+        public Data $data = new KeyedDataObject(),
```

Comment:
I would expect this to be private, since we cannot type properties in interfaces (yet).